Example/SwiftUIExample/MobilePaymentsSwiftUIExample.xcodeproj/project.pbxproj

```diff
@@ -0,0 +1,558 @@
+				FB1E68AE2C1CC50A004DC0A9 /* Products */,
+			);
+			sourceTree = "<group>";
+		};
+		FB1E68AE2C1CC50A004DC0A9 /* Products */ = {
+			isa = PBXGroup;
+			children = (
+				FB1E68AD2C1CC50A004DC0A9 /* MobilePaymentsSwiftUIExample.app */,
+			);
+			name = Products;
+			sourceTree = "<group>";
+		};
+		FB1E68AF2C1CC50A004DC0A9 /* MobilePaymentsSwiftUIExample */ = {
+			isa = PBXGroup;
+			children = (
+				D09D32302CE4703B002DB265 /* Helpers */,
+				D0D5BB632CDD75DE0054771D /* Extensions */,
+				FB1E68C42C1CC55C004DC0A9 /* Components */,
+				FB1E68C32C1CC55C004DC0A9 /* MobilePaymentsApp.swift */,
+				FB1E68C72C1CC55C004DC0A9 /* Screens */,
+				FB1E68DF2C1CC5E8004DC0A9 /* Shared */,
+			);
+			path = MobilePaymentsSwiftUIExample;
+			sourceTree = "<group>";
+		};
+		FB1E68C42C1CC55C004DC0A9 /* Components */ = {
+			isa = PBXGroup;
+			children = (
+				FB1E68C52C1CC55C004DC0A9 /* MobilePaymentsSDKViewHolder.swift */,
+				FB1E68C62C1CC55C004DC0A9 /* AppButtons.swift */,
+			);
+			path = Components;
+			sourceTree = "<group>";
+		};
+		FB1E68C72C1CC55C004DC0A9 /* Screens */ = {
+			isa = PBXGroup;
+			children = (
+				D0F1F9262CDEB44700E00BB6 /* Permissions */,
+				D0F1F9252CDEB44200E00BB6 /* Home */,
+			);
+			path = Screens;
+			sourceTree = "<group>";
+		};
+		FB1E68DF2C1CC5E8004DC0A9 /* Shared */ = {
+			isa = PBXGroup;
+			children = (
+				FB1E68E02C1CC5E8004DC0A9 /* Resources */,
+				FB1E68E32C1CC5E8004DC0A9 /* Config.swift */,
+				FB1E68E42C1CC5E8004DC0A9 /* Preview Content */,
+			);
+			name = Shared;
+			path = ../../Shared;
+			sourceTree = "<group>";
+		};
+		FB1E68E02C1CC5E8004DC0A9 /* Resources */ = {
+			isa = PBXGroup;
+			children = (
+				FB1E68E12C1CC5E8004DC0A9 /* Assets.xcassets */,
+				FB1E68E22C1CC5E8004DC0A9 /* Launch Screen.storyboard */,
+			);
+			path = Resources;
+			sourceTree = "<group>";
+		};
+		FB1E68E42C1CC5E8004DC0A9 /* Preview Content */ = {
+			isa = PBXGroup;
+			children = (
+				FB1E68E52C1CC5E8004DC0A9 /* Preview Assets.xcassets */,
+				FB1E68E62C1CC5E8004DC0A9 /* MockMobilePaymentsSDK.swift */,
+			);
+			path = "Preview Content";
+			sourceTree = "<group>";
+		};
+/* End PBXGroup section */
+
+/* Begin PBXNativeTarget section */
+		FB1E68AC2C1CC50A004DC0A9 /* MobilePaymentsSwiftUIExample */ = {
+			isa = PBXNativeTarget;
+			buildConfigurationList = FB1E68BB2C1CC50A004DC0A9 /* Build configuration list for PBXNativeTarget "MobilePaymentsSwiftUIExample" */;
+			buildPhases = (
+				FB1E68A92C1CC50A004DC0A9 /* Sources */,
+				FB1E68AA2C1CC50A004DC0A9 /* Frameworks */,
+				FB1E68AB2C1CC50A004DC0A9 /* Resources */,
+				FB1E68EC2C1CC5FB004DC0A9 /* ShellScript */,
+			);
+			buildRules = (
+			);
+			dependencies = (
+			);
+			name = MobilePaymentsSwiftUIExample;
+			packageProductDependencies = (
+				FB1E68DB2C1CC58C004DC0A9 /* MockReaderUI */,
+				FB1E68DD2C1CC58C004DC0A9 /* SquareMobilePaymentsSDK */,
+				D05023652CE52A5B005FA05C /* MockReaderUI */,
```

Comment:
I wonder why there are two references to each dependency

Reply:
Seems like it cached the local SPM sdk as well as the remote SPM sdk. Was able to remove it.